src/Tribe/Repositories/Event.php

```diff
@@ -263,10 +263,13 @@ public function filter_by_starts_before( $datetime, $timezone = null ) {
 		$date = Tribe__Date_Utils::build_date_object( $datetime, $timezone )
 		                         ->setTimezone( $this->normal_timezone );
 
+		// If this is a UTC date, use our UTC field, else whatever was specified.
+		$key = $date->getTimezone()->getName() === 'UTC' ? '_EventStartDateUTC' : $this->start_meta_key;
```

Comment:
We do this 8+ times. In most instances we are setting the timezone just before. Does it make sense to just check `$this->normal_timezone` instead?

Feel like I'm missing something here...

Reply:
I guess most of them could be replaced with that... half dozen of one, six of the other. Not sure there's really a big difference, except for a couple places it was made more complicated needlessly.